### DISCUSSION TITLE
Problems when build

### DISCUSSION THREAD

**Discussion by hocdev** [1Panel-dev/1Panel#5519] [Q&A]
**chrome console**

index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.contain' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.equal' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.regex' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.notEqual' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.badReq' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.forbidden' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.noRes' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.serverErr' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.actionAllow' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.deny' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.captcha' key in 'zh' locale messages.
index-Dhd6qK6j.js:79 [intlify] Not found 'xpack.waf.fiveSeconds' key in 'zh' locale messages.

Error messages when I build the version myself

and title is " loading..." . How to change it?
![Untitled](https://github.com/1Panel-dev/1Panel/assets/44840759/e5fe0f60-eeac-496e-9cdf-bf2deebeeb99)

**Comment by zhengkunwang223** [comment]
这个有可能是你引用的问题 你检查一下有没有 xpack.waf 开头的翻译
